conans/test/unittests/client/build/msbuild_test.py

```diff
@@ -131,17 +133,6 @@ def get_version_test(self):
         self.assertRegexpMatches(version, "(\d+\.){2,3}\d+")
         self.assertGreater(version, "15.1")
 
-    def custom_properties_test(self):
```

Comment:
Why removing this test?

Reply:
It's defined twice. With the same name and the same body.

Reply:
See https://github.com/conan-io/conan/blob/b70712772e6d6615750b615f4cb3d4b472b9c273/conans/test/unittests/client/build/msbuild_test.py#L48